src/Mod/Start/Gui/FileCardDelegate.cpp

```diff
@@ -49,6 +50,32 @@ FileCardDelegate::FileCardDelegate(QObject* parent)
         "User parameter:BaseApp/Preferences/Mod/Start");
 }
 
+QColor FileCardDelegate::getBorderColor() const
+{
+    QColor color(98, 160, 234);  // NOLINT
+    uint32_t packed = App::Color::asPackedRGB<QColor>(color);
+    packed = _parameterGroup->GetUnsigned("FileThumbnailBorderColor", packed);
+    color = App::Color::fromPackedRGB<QColor>(packed);
+    return color;
+}
+
+QColor FileCardDelegate::getBackgroundColor() const
+{
+    QColor color(221, 221, 221);  // NOLINT
+    uint32_t packed = App::Color::asPackedRGB<QColor>(color);
+    packed = _parameterGroup->GetUnsigned("FileThumbnailBackgroundColor", packed);
+    color = App::Color::fromPackedRGB<QColor>(packed);
+    return color;
+}
+
+QColor FileCardDelegate::getSelectionColor() const
+{
+    QColor color(38, 162, 105);  // NOLINT
+    uint32_t packed = App::Color::asPackedRGB<QColor>(color);
+    packed = _parameterGroup->GetUnsigned("FileThumbnailSelectionColor", packed);
+    color = App::Color::fromPackedRGB<QColor>(packed);
+    return color;
+}

```

Comment:
can't we default to reasonable colors from the main window palette?

Reply:
These colours are used if no application-wide style sheet is set. To make it working when using a style sheet then the .qss files must be extended: see  https://github.com/FreeCAD/FreeCAD/pull/13604?notification_referrer_id=NT_kwDOAGxdLbMxMDM3NjY0ODI5ODo3MTAxNzQx&notifications_query=is%3Aunread#issuecomment-2074393988

Reply:
Indeed I'll be working on adding support to my stylesheets in the next day or two. I do find it interesting though that QToolButtons were not used here which would have observed the stylesheets by default, but as a layman, I assume there was a good reason for this approach instead.